docs/pages/core-concepts.mdx

```diff
@@ -78,27 +106,24 @@ PostgreSQL and MySQL.
 Read more about the [Teleport Database
 Service](./enroll-resources/database-access/database-access.mdx).
 
+### Teleport Discovery Service
+
+The `discovery_service` automates the process of finding and enrolling cloud resources such as AWS EC2 instances and Azure VMs into a Teleport cluster. It continuously scans supported cloud environments and adds matching resources based on configured rules, reducing the need for manual node registration. Read more about the [Teleport Discovery Service](./enroll-resources/auto-discovery/auto-discovery.mdx).
```

Comment:
- I would split up the paragraph into multiple lines to make editing easier.
- I would use "Teleport Discovery Service" instead of the configuration field name (same for other service names), since this paragraph describes the service and not just the field.